### DISCUSSION TITLE
Switching between groups

### DISCUSSION THREAD

**Discussion by haithium** [qtile/qtile#4661] [Q&A] [answered]
How do I switch between groups using group names in the command line? Can I use something like `lazy.group["group-name"].toscreen()`?

**Comment by elParaguayo** [comment, marked as answer]
`qtile cmd-obj -o group group-name -f toscreen`

**Reply by haithium** [reply]
That's it, I feel so dumb. Thank you.

**Reply by elParaguayo** [reply]
Don't feel bad. The `cmd-obj` interface isn't as well known. Once you're familiar with it, it's easy to translate `lazy` commands to `qtile cmd-obj`.
